src/drc/api/views/enkelvoudig_informatieobject.py

```diff
@@ -208,7 +225,7 @@ def get_serializer_class(self):
         """
         if self.action in ["update", "partial_update"]:
             return EnkelvoudigInformatieObjectWithLockSerializer
-        elif self.action == "create":
+        if self.action == "create":
```

Comment:
```suggestion
        elif self.action == "create":
```

Deze wijziging is volgens mij overbodig